include/vinecopulib/vinecop/implementation/tools_select.ipp

```diff
@@ -332,9 +337,16 @@ inline FamilySelector::FamilySelector(const Eigen::MatrixXd &data,
 }
 
 inline StructureSelector::StructureSelector(const Eigen::MatrixXd &data,
-                                            const FitControlsVinecop &controls) 
+                                            const FitControlsVinecop &controls)
     : VinecopSelector(data, controls)
 {
+    n_ = data.rows();
```

Comment:
This stuff is already done in the `VinecopSelector` constructor.

Reply:
true